smartseq2_single_sample/ss2_single_sample_kco.wdl

```diff
@@ -0,0 +1,165 @@
+task Star{
+  File input_fastq_read1
+  File input_fastq_read2
+  File gtf
+  File  starGenome
+  command {
+    tar -zxvf ${starGenome}
+    STAR  --readFilesIn ${input_fastq_read1} ${input_fastq_read2} --genomeDir ./star  --quantMode TranscriptomeSAM   --outSAMstrandField intronMotif --genomeLoad NoSharedMemory --sjdbGTFfile ${gtf} --readFilesCommand "zcat" --twopassMode Basic --outSAMtype BAM SortedByCoordinate  --outSAMunmapped Within --limitBAMsortRAM 30000000000 
+  }
+  output {
+    File junction_table = "SJ.out.tab"
+    File output_bam = "Aligned.sortedByCoord.out.bam"
+    File output_bam_trans="Aligned.toTranscriptome.out.bam"
+  }
+  runtime{
+    docker:"humancellatlas/star_dev:v1"
+    memory: "40 GB"
+    disks :"local-disk 100 HDD"
+  }
+}
+task FeatureCountsUniqueMapping {
+  File aligned_bam
+  File gtf
+  String fc_out
+  command{
+    featureCounts -s 0 -t exon -g gene_id -p -B -C -a ${gtf} -o "${fc_out}.gene.counts.txt" ${aligned_bam}
+    featureCounts -s 0 -t exon -g transcript_id -p -B -C -a ${gtf} -o "${fc_out}.transcripts.counts.txt" ${aligned_bam}
+    featureCounts -s 0 -t exon -g exon_id -p -B -C -a ${gtf} -o "${fc_out}.exon.counts.txt" ${aligned_bam}
+  }
+ runtime{
+    docker:"humancellatlas/star_dev:v1"
+    memory: "10 GB"
+    disks :"local-disk 50 HDD"
+  }
+ output {
+    File genes="${fc_out}.gene.counts.txt"
+    File exons="${fc_out}.exon.counts.txt"
+    File trans="${fc_out}.transcripts.counts.txt"
+  }
+}
+task rsem_expression{
+  File trans_aligned_bam
+  File rsemGenome
+  String rsem_out
+  command{
+    tar -zxvf ${rsemGenome}
+    echo "Aligning fastqs and calculating expression"
+    rsem-calculate-expression --bam --paired-end ${trans_aligned_bam} rsem/rsem_trans_index  "${rsem_out}"
+    cut -f 1,4,5 "${rsem_out}.genes.results" >"${rsem_out}.gene.expected_counts"
+  }
+  runtime{
+    docker: "humancellatlas/rsem"
+    memory: "10 GB"
+    disks: "local-disk 100 HDD"
+  }
+  output{
+    File rsem_gene="${rsem_out}.genes.results"
+    File rsem_transc="${rsem_out}.isoforms.results"
+    File rsem_gene_count="${rsem_out}.gene.expected_counts"
+   }
+}
+task CollectAlignmentSummaryMetrics{
```

Comment:
space after `CollectAlignmentSummaryMetrics` and throughout as mentioned above, spaces before all curly braces basically